playbooks/common/openshift-cluster/upgrades/pre/version_override.yml

```diff
@@ -0,0 +1,26 @@
+---
+# This playbook overrides normal version setting during double upgrades.
+
+- name: Set proper version values for upgrade
+  hosts: "{{ l_version_override_hosts | default('all:!all') }}"
+  tasks:
+    - set_fact:
+        # All of these will either have been set by openshift_version or
+        # provided by the user; we need to save these for later.
+        l_double_upgrade_saved_version: "{{ openshift_version }}"
+        l_double_upgrade_saved_release: "{{ openshift_release }}"
+        l_double_upgrade_saved_tag: "{{ openshift_image_tag }}"
+        l_double_upgrade_saved_pkgv: "{{ openshift_pkg_version | default('') }}"
```

Comment:
Right, openshift_pkg_version is completely unused in atomic hosts.  For RPM hosts, it will be set to something.  I'm defaulting it to an empty string to hard-code it's value to empty string.

Reply:
Hmm, looking at version's source, it should be setting this.  I was hitting this as undefined for some reason before; no need for this default.